### ISSUE TITLE
Throwing Exception

### ISSUE THREAD

**Issue by mikebronner** [beyondcode/laravel-xray#5]
#### Stack Trace
```[2018-07-19 12:33:50] development.ERROR: Call to a member function getPath() on null {"userId":17,"email":"mike@genealabs.com","exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getPath() on null at /home/vagrant/Sites/ensdevc.com/vendor/beyondcode/laravel-view-xray/src/XrayMiddleware.php:102)
[stacktrace]
#0 /home/vagrant/Sites/ensdevc.com/vendor/beyondcode/laravel-view-xray/src/XrayMiddleware.php(47): BeyondCode\\ViewXray\\XrayMiddleware->injectXrayBar(Object(Illuminate\\Http\\Response))
#1 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): BeyondCode\\ViewXray\\XrayMiddleware->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#2 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#3 /home/vagrant/Sites/ensdevc.com/vendor/beyondcode/laravel-query-detector/src/QueryDetectorMiddleware.php(33): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#4 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): BeyondCode\\QueryDetector\\QueryDetectorMiddleware->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#5 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#6 /home/vagrant/Sites/ensdevc.com/vendor/barryvdh/laravel-debugbar/src/Middleware/InjectDebugbar.php(65): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#7 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Barryvdh\\Debugbar\\Middleware\\InjectDebugbar->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#8 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#9 /home/vagrant/Sites/ensdevc.com/vendor/genealabs/laravel-casts/src/Http/Middleware/AssetInjection.php(10): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#10 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): GeneaLabs\\LaravelCasts\\Http\\Middleware\\AssetInjection->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#11 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#12 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(30): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#13 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#14 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#15 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(30): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#16 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#17 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#18 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#19 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#20 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#21 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#22 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode->handle(Object(Illuminate\\Http\\Request), Object(Closure))
#23 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Illuminate\\Http\\Request))
#24 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}(Object(Illuminate\\Http\\Request))
#25 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(151): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#26 /home/vagrant/Sites/ensdevc.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#27 /home/vagrant/Sites/ensdevc.com/public/index.php(53): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#28 {main}
```

**Comment by mpociot** [comment]
Can you tell me what kind of response you’re doing? Seems like the middleware is unable to inject the HTML for the X-ray bar. 
And what’s the actual exception message?

**Comment by mikebronner** [comment]
Weird thing is that I can't recreate the problem anymore. The response would have been a View response. Sorry about the exception message ... looks like it got truncated when I copied it. :(

I'll re-open if it comes up again.

**Comment by mikebronner** [comment]
@mpociot  So I was able to get it to re-occur:
<img width="1208" alt="screen shot 2018-07-19 at 12 02 43 pm" src="https://user-images.githubusercontent.com/1791050/42964407-c720134e-8b4b-11e8-863d-67232e4043e5.png">

**Update:** it looks like this happens whenever the app throws an exception for another reason.

**Comment by mpociot** [comment]
Thanks, Mike.
I fixed this in v1.0.2
